### PR TITLE
Update analytics session events to include app & auth type info

### PR DESCRIPTION
**Pull request by acrollet**
## Description

We need to be able to track session authentication success with SSN4 & DOB validation. This PR updates the API analytics event labels to include the app name and auth type.

## Original issue(s)
department-of-veterans-affairs/va.gov-team#44383

## Acceptance criteria
- [ ] event labels include day-of/precheckin and ssn4/dob

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/check-in/api/versions/v2.js [hunk 1]
```diff
@@ -3,12 +3,20 @@ import environment from 'platform/utilities/environment';
...
 const v2 = {
-  getSession: async ({ token, checkInType }) => {
+  getSession: async ({
+    token,
+    checkInType,
+    isLorotaSecurityUpdatesEnabled = false,
+  }) => {
     const url = `/check_in/v2/sessions/`;
     const checkInTypeSlug = checkInType ? `?checkInType=${checkInType}` : '';
+    const eventLabel = `${checkInType ?? 'day-of'}-get-current-session-${
```
Comment (brianseek) [inline src/applications/check-in/api/versions/v2.js:13]: is the checkInType stuff here redundant of whats already in the slug>

Comment (acrollet) [reply src/applications/check-in/api/versions/v2.js:13]: hmm I don't think so, this is what I see in GA currently:
Screen Shot 2022-07-21 at 10 15 46

Comment (acrollet) [reply src/applications/check-in/api/versions/v2.js:13]: and: 
Screen Shot 2022-07-21 at 10 16 25

Comment (brianseek) [reply src/applications/check-in/api/versions/v2.js:13]: ah ok, these things translate weird into GA

**Review by brianseek — APPROVED** [review]
LGTM!

**Pull request merged**